autoscheduler/frontend/src/tests/ui/CourseSelectColumn.test.tsx

```diff
@@ -22,20 +22,20 @@ beforeEach(() => {
   document.body.innerHTML = '';
 });
 
-function ignoreInvisible(query: string | RegExp):
-(content: string, element: HTMLElement) => boolean {
-  return (content: string, element: HTMLElement): boolean => {
-    if (content.match(query) && content.match(query).length > 0) {
-      try {
-        expect(element).toBeVisible();
-        return true;
-      } catch {
-        return false;
-      }
-    }
-    return false;
-  };
-}
+// function ignoreInvisible(query: string | RegExp):
+// (content: string, element: HTMLElement) => boolean {
+//   return (content: string, element: HTMLElement): boolean => {
+//     if (content.match(query) && content.match(query).length > 0) {
+//       try {
+//         expect(element).toBeVisible();
+//         return true;
+//       } catch {
+//         return false;
+//       }
+//     }
+//     return false;
+//   };
+// }
```

Comment:
Make sure to remove unused code blocks like this instead of commenting them out when you commit